added email sending to new user

memeraki opened the pull request.

memeraki committed:
added email sending to new user

AleksandraCyp commented on src/index.ts:
```diff
@@ -8,7 +8,7 @@ if (!process.env.JWT_PRIVATE_KEY) {
 }
 
-mongoose.connect(`mongodb+srv://${process.env.DB_USER}:${process.env.DB_PASSWORD}@cluster0.svu8r.mongodb.net/Pikabook?retryWrites=true&w=majority`, {useNewUrlParser: true, useCreateIndex:true, useUnifiedTopology: true}, () => {
+mongoose.connect(`mongodb://localhost:27017/?readPreference=primary&appname=MongoDB%20Compass&ssl=false`, {useNewUrlParser: true, useCreateIndex:true, useUnifiedTopology: true}, () => {
```
why do you change it to localhost?

memeraki committed:
edited db path and some interface/schema names

I edited the interface and schema a bit for more consistent names, because I noticed that we used the name User and userModel interchangeably (same for books).

memeraki committed:
error fix
